### DISCUSSION TITLE
Alerts schedule

### DISCUSSION THREAD

**Discussion by orion131313** [roleoroleo/yi-hack-Allwinner-v2#426] [Q&A]
Hi,
Is it possible to configure a schedule so that the camera gives movement alerts only in a certain period (I only want it during the night when it detects a human) as in the YI app?

**Comment by WalkingDot** [comment]
The webinterface has a crontab option.
Something like this should work:
`0 6 * * * /tmp/sd/yi-hack/bin/ipc_cmd -t off > /dev/null 2>&1`
`0 22 * * * /tmp/sd/yi-hack/bin/ipc_cmd -t on > /dev/null 2>&1`

**Reply by orion131313** [reply]
Hi,
it works
Thanks

**Reply by orion131313** [reply]
Hi,
This command turns off the camera. I do not want it. I just want it to turn off alert events. Is there any command for this?

**Reply by WalkingDot** [reply]
You mean turn off the MQTT service? I think this should also work... but I don't know the command.

**Reply by orion131313** [reply]
No, I don't use the MQTT service.
I want to turn off/on the alerts caused by human detection to record or not the video on the sd card.

**Comment by WalkingDot** [comment]
The args for ai_human_detection are:
ipc_cmd -a off
ipc_cmd -a on

**Comment by orion131313** [comment]
With this option AIHUMANDETECTION OFF I will receive all movement alerts and not just humans. I want to cancel all movement alerts in a certain period.
I believe this is not possible in YI-Hack firmware. On the configuration page it indicates that the option to turn on/off alerts is only available in the Yi app. In the app, if I put ON, I have a submenu where I can define a cronotab. Yi-hack doesn't recognize this cronotab.
Thanks

**Reply by roleoroleo** [reply]
> With this option AIHUMANDETECTION OFF I will receive all movement alerts and not just humans. I want to cancel all movement alerts in a certain period.

There is no way to cancel motion detection from yi-hack. I didn't find a command to obtain this behavior.

> In the Yi app. In the app, if I put ON, I have a submenu where I can define a cronotab. Yi-hack doesn't recognize this cronotab.

This is a custom crontab and it's not available. The hack can't use it.
